Description: Fix shebang in make-pluginlist.sh

Author: Yury Usishchev <y.usishchev@contactless.ru>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- lirc-0.9.3a.orig/plugins/make-pluginlist.sh
+++ lirc-0.9.3a/plugins/make-pluginlist.sh
@@ -1,4 +1,4 @@
-#/bin/bash
+#!/bin/bash
 #
 # Create a list of automake module setups.
 
